runbotics/runbotics-scheduler/src/database/process/process.service.ts

```diff
@@ -4,7 +4,17 @@ import { Repository } from 'typeorm';
 import { ProcessEntity } from './process.entity';
 import { IProcess } from 'runbotics-common';
 
-const relations = ['createdBy', 'system', 'botCollection', 'schedules', 'editor', 'notifications.user.authorities', 'credentials.attributes'];
+const relations = [
+    'createdBy',
+    'system',
+    'botCollection',
+    'schedules',
+    'editor',
+    'notifications.user.authorities',
+    'credentials.attributes',
+    'credentials.attributes.secret',
+    'credentials.template',
```

Comment:
to rethink - maybe we want to use credentials.attributes.secret only where we need it - not everytime we GET process

Reply:
we want to keep both encrypted and decrypted credentials on sheduler only, and never send them to UI or keep em on the endpoint

Reply:
only to send decrypted to bot when it's supposed to run new process